core/connection/manager.go

```diff
@@ -417,6 +417,22 @@ func (manager *connectionManager) onStateChanged(state State) {
 	}
 }
 
+func (manager *connectionManager) setupTrafficBlock(disableTrafficBlocking bool) error {
```

Comment:
`DisableKillSwitch`, not worth to have 2 different namings

Reply:
I am note sure about DisableKillSwitch term - its really vague and doesn't tell anything. Didn't want to modify tequilapi yet, but would go for BlockNonTunneledTraffic property coming from API.

Reply:
It's ordinary naming in VPN domain

Reply:
Show me

Reply:
anyway left old naming